src/data/services/LmsApiService.js

```diff
@@ -41,7 +41,9 @@ class LmsApiService {
 
   static enterpriseSsoOrchestrationUrl = `${LmsApiService.baseUrl}/enterprise/api/v1/enterprise_customer_sso_configuration/`;
 
-  static enterpriseGroupUrl = `${LmsApiService.baseUrl}/enterprise/api/v1/enterprise_group/`;
+  static enterpriseGroupUrl = `${LmsApiService.baseUrl}/enterprise/api/v1/enterprise-group/`;
+
+  static enterpriseGroupListUrl = `${LmsApiService.baseUrl}/enterprise/api/v1/enterprise_group/`;
```

Comment:
Do we really have two api's only disambiguated by dash/underscore?

Reply:
yeah... there are inconsistencies on the backend how the urls are defined. We should probably have a ticket to clean them up